### ISSUE TITLE
VOD chats no longer works on newest version

### ISSUE THREAD

**Issue by ScalableCodex** [twireapp/Twire#374]
Just updated to the newest version on F-Droid (2.10.4) and getting chat to work with VODs basically never works anymore. Worked perfectly fine before on 2.10.3.

**Comment by samfundev** [comment]
I fixed a bug where the VOD chat would stop before the end of the VOD: 4f9f34fb0be6b3d1ce72c6e0b0d6cb41e265134c
Is that what you are experiencing?

**Comment by ScalableCodex** [comment]
No, its weird. I can't recreate it every time, as it happens at random. But what SEEMS to trigger it most of the time, is if I skip around. Or when I start the VOD. 

Sometimes when I start the VOD, the comments won't load at all. But sometimes they will. Going back, and restarting it will usually fix it, but not always. Sometimes it takes a few reloads. 

And same with when I skip around the VOD, either through the time skip buttons, or if I use the slider. Usually reloading the VOD will fix it too. But not always the first time.

**Comment by samfundev** [comment]
Hmm, well you can try [this build](https://github.com/twireapp/Twire/suites/8230441114/artifacts/359186792) out which should have the fixes in it. If that works, then your issue should be fixed in the next version.
